src/index.html

```diff
@@ -4,13 +4,57 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+      >
+      <label>
+        <div class="bigBox">
+          <img
+            class="bigIcon"
+            src="./images/Search.svg"
+            alt="search"
+            >
```

Comment:
```suggestion
          <img
            class="bigIcon"
            src="./images/Search.svg"
            alt="search"
          >
```
you need to place this bracket on the same level as open one, fix it everywhere